packages/plugin-webpack/src/plugin.ts

```diff
@@ -0,0 +1,32 @@
+import { postProcess, PostProcessOptions } from '@metamask/snap-utils';
+import { Compiler, WebpackPluginInstance } from 'webpack';
+import { RawSource } from 'webpack-sources';
+
+const PLUGIN_NAME = 'SnapWebpackPlugin';
```

Comment:
To add further context for this: the branding we've landed on is "MetaMask Snaps", so we should use "Snaps" in the name for packages / things intended for external audiences:

```suggestion
const PLUGIN_NAME = 'SnapsWebpackPlugin';
```